src/main/java/beans/debugger/QuestionResponseItem.java

```diff
@@ -11,6 +11,12 @@
     private String label;
     private String type;
 
+    public QuestionResponseItem(String value, String label, String type) {
```

Comment:
this might make sense to refactor to implement XPathAutoCompleteItem or something so that it makes more semantic sense